### ISSUE TITLE
sample raspi streaming doesn't work

### ISSUE THREAD

**Issue by KeisukeNagakawa** [nttcom/skyway-iot-sdk#1]
Hi, I'm trying to  install skyway-iot-sdk on raspi3 with raspi-camera module, but failed to see the test video stream and realtime metrics on this sample web page.

What I did  is as below: 
```
/opt/janus/bin/janus
node app 
bash h264_streaming_raspicamm.sh
node metics
```
And error occures in "node app" command.
Error is as follows: 

> [WARN] ProfileManager - FetchError: request to http://localhost:5001/ssg_peerid failed, reason: connect ECONNREFUSED 127.0.0.1:5001
> (node:1739) UnhandledPromiseRejectionWarning: Unhandled promise rejection (rejection id: 2): FetchError: invalid json response body at http://localhost:5000/profile reason: Unexpected token F in JSON at position 0
> (node:1739) [DEP0018] DeprecationWarning: Unhandled promise rejections are deprecated. In the future, promise rejections that are not handled will terminate the Node.js process with a non-zero exit code.
> (node:1739) UnhandledPromiseRejectionWarning: Unhandled promise rejection (rejection id: 3): TypeError: Cannot read property 'isLevelEnabled' of undefined
> 

And of course no streaming is shown in the sample web page: 
https://nttcom.github.io/skyway-siru-sample/dist/

Message is shown like this:

```
waiting for devices...
lastUpdated: 0
```

I attached the detailed logs for each command!
[metrics-command-log.txt](https://github.com/nttcom/skyway-iot-sdk/files/1082242/metrics-command-log.txt)
[node-app-command-log.txt](https://github.com/nttcom/skyway-iot-sdk/files/1082243/node-app-command-log.txt)
[raspi-stream-command-log.txt](https://github.com/nttcom/skyway-iot-sdk/files/1082244/raspi-stream-command-log.txt)
[janus-command-log.txt](https://github.com/nttcom/skyway-iot-sdk/files/1082245/janus-command-log.txt)

cf. I also tried h264_streaming_testsrc.sh instead of bash h264_streaming_raspicamm.sh  but it didn't succeed. error is as follows.
```
Caught SIGSEGV
#0  0x76bc7b80 in poll () at ../sysdeps/unix/syscall-template.S:81
#1  0x76cb2528 in ?? () from /lib/arm-linux-gnueabihf/libglib-2.0.so.0
Spinning.  Please run 'gdb gst-launch-1.0 13225' to continue debugging, Ctrl-C to quit, or Ctrl-\ to dump core.
^Chandling interrupt.
Interrupt: Stopping pipeline ...
ERROR: pipeline doesn't want to preroll.
```

Thanks

**Comment by KensakuKOMATSU** [comment]
Apporogize for late reply and inconvenience.

I'll check and fix this issue by next week. Please wait for a while.

**Comment by KensakuKOMATSU** [comment]
I fixed this issue on [v0.5.3](https://github.com/nttcom/skyway-signaling-gateway/releases/tag/v0.5.3). Could you update [skyway-signaling-gateway](https://github.com/nttcom/skyway-signaling-gateway) to latest one?

I suppose your trouble is due to connection issue to skyway signaling server. However, previous version did not make any useful log message to fix it.

Current version shows you up a proper message while some trouble happens for connecting servers. I think it will work well for addressing the trouble.

And more, I fixed the gstreamer shell script for ``h264_streaming_testsrc.sh``. I hope it will work for you.
